backend/plugin/db/databricks/sync.go

```diff
@@ -0,0 +1,181 @@
+package databricks
+
+import (
+	"context"
+	"time"
+
+	"github.com/databricks/databricks-sdk-go/service/catalog"
+	"github.com/pkg/errors"
+
+	"github.com/bytebase/bytebase/backend/plugin/db"
+	"github.com/bytebase/bytebase/proto/generated-go/store"
+)
+
+type storepbTable struct {
```

Comment:
Why it's called storepbTable? Could you find a better name?

Reply:
Maybe 'TableUnion' is better?